package/wpe/wpeframework-plugins/wpeframework-plugins.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_PLUGINS_VERSION = 54bec7aff70825500c49ece273f871e4028b8d7f 
+WPEFRAMEWORK_PLUGINS_VERSION = 2039b2c6cfcecbb159f5289ba923c8a67dd9347b
```

Comment:
Please drop the bumps, I think they point to another branch?

Reply:
I will change this into hash of master branch